libcodechecker/server/session_manager.py

```diff
@@ -4,38 +4,38 @@
 #   License. See LICENSE.TXT for details.
 # -------------------------------------------------------------------------
 """
-Handles the allocation and destruction of privileged sessions associated
-with a particular CodeChecker server.
+Handles the management of authentication sessions on the server's side.
 """
 
 from datetime import datetime
 import hashlib
-import json
 import os
 import shutil
-import stat
 import time
 import uuid
 
-import portalocker
-
 from libcodechecker.logger import get_logger
+<<<<<<< HEAD:libcodechecker/session_manager.py
```

Comment:
Could you remove these git marks from the source?

Reply:
Done these too.